docs/pages/access-controls/guides/webauthn.mdx

```diff
@@ -235,82 +242,411 @@ $ tsh login --proxy=mytenant.teleport.sh
+Device <Var name="(=devicetrust.asset_tag=)"/>/macos added to the inventory
+```
+
+Use `tctl` to check that the device has been registered:
+
+```code
+$ tctl devices ls
+Asset Tag    OS    Enroll Status Device ID
+------------ ----- ------------- ------------------------------------
+(=devicetrust.asset_tag=) macOS not enrolled  (=devicetrust.device_id=)
+```
+
+## Step 2/3. Create a device enrollment token
+
+A registered device becomes a trusted device after it goes through the
+enrollment ceremony. To enroll the device, a device enrollment token is
+necessary. The token is created by a device admin and sent to the person
+performing the enrollment off-band (for example, via a corporate chat).
+
+To create an enrollment token run the command below, where `--asset-tag` is
+the serial number of the device we want to enroll:
+
+```code
+$ tctl devices enroll --asset-tag="(=devicetrust.asset_tag=)"
+Run the command below on device "(=devicetrust.asset_tag=)" to enroll it:
+tsh device enroll --token=(=devicetrust.enroll_token=)
+```
+
+## Step 3/3. Enroll a trusted device
+
+To perform the enrollment ceremony, using the device specified above, type the
+command printed by `tctl devices enroll`:
+
+```code
+$ tsh device enroll --token=(=devicetrust.enroll_token=)
+Device "(=devicetrust.asset_tag=)"/macOS enrolled
+
+$ tsh logout
+$ tsh login --proxy=(=clusterDefaults.clusterName=) --user=(=clusterDefaults.username=) # fetch new certificates
+Enter password for Teleport user (=clusterDefaults.username=):
+Tap any security key
+Detected security key tap
+> Profile URL:        (=clusterDefaults.clusterName=):443
+  Logged in as:       (=clusterDefaults.username=)
+  Cluster:            (=clusterDefaults.clusterName=)
+  Roles:              access, editor
+  Logins:             (=clusterDefaults.username=)
+  Kubernetes:         enabled
+  Valid until:        2023-06-23 02:47:05 -0300 -03 [valid for 12h0m0s]
+  Extensions:         teleport-device-asset-tag, teleport-device-credential-id, teleport-device-id
+```
+The presence of the `teleport-device-*` extensions shows that the device was
+successfully enrolled and authenticated. The device above is now a trusted device.
+
+### Auto-Enrollment
+
+Distributing enrollment tokens to many users can be challenging. To address that,
+Teleport supports auto-enrollment. When enabled, auto-enrollment automatically
+enrolls the user's device in their next Teleport (`tsh`) login.
+
+For auto-enrollment to work, the following conditions must be met:
+- A device must be registered. Registration may be [manual](#implement-cluster-wide-device-trust) or performed using an
+integration, like the [Jamf Pro integration](../device-trust/jamf-integration.mdx).
+- Auto-enrollment must be enabled in the cluster setting.
+- User must have either preset `editor` or `device-enroll` (available v13.3.6+) role assigned to them.
+
+Enable auto-enrollment in your cluster settings:
+
+<Tabs dropDownCaption="Teleport Deployment">
```

Comment:
I would avoid using a complex tabs component here and just show the "Dynamic Resources" example.

Reply:
Agreed, that looks better! I can open an issue to clear all of these from the guide.